### PR TITLE
Update current team + add members to past teams

### PR DESCRIPTION
**Pull request by jcserv**
## feature/update_teams_data

- [ ] Have dependencies/packages been added/removed? If so, please list them.

### Changes

List out all the changes, and then explain the changes made at a higher level

* Update current team data
* Add previous year's team
* Add personal site link if provided
* Change Discord icon to conditional render if field provided
* Remove Tech of The Month from NavDrawer + Footer

Misc.
* Add PULL_REQUEST_TEMPLATE.md
* Delete unused files (FemaleLion.png, Lion.png)
* Remove .vscode

### CODE REVIEW

File: content/team/team.json [hunk 1]
```diff
@@ -1,58 +1,52 @@
...
       "email": "nameera.azim@mail.utoronto.ca",
       "linkedin": "https://www.linkedin.com/in/nameera-azim-1924a21b6/"
     },
+    {
+      "name": "Naaz Sibia",
+      "imgSrc": "/team/2021/Naaz.png",
+      "position": "Director of Operations",
+      "message": "Naaz likes preference learning and Human-Computer Interaction (with a CS-ed focus). She also likes talking about mental health, and courses she's TA'ed. When not looking into the mentioned topics, or working to finish her bachelor's degree, she likes having 3 am discussions with friends, drawing chibis, and eating Chinese food.",
+      "discord": "Blossom#1234",
+      "email": "vyvy.ngo@mail.utoronto.ca",
```
Comment (hiimchrislim) [inline content/team/team.json:27]: Might want to change this to Naaz' email and Linkedin 😅

**Review by hiimchrislim — CHANGES_REQUESTED** [review]
Looks good although Naaz' email and Linkedin might want to be updated. It still has VyVy's info in there

File: content/team/team.json (hunk 1 above)
Comment (jcserv) [reply content/team/team.json:27]: Thanks haha, can’t believe I missed that

**Review by ander710 — APPROVED** [review]

**Review by hiimchrislim — APPROVED** [review]
Looks great!

**Pull request merged**